src/app/__tests__/TopicDashboard/index.test.js

```diff
@@ -1,9 +1,19 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import TopicDashboard from '../../TopicDasboard';
+import { TopicDashboard } from '../../TopicDasboard';
+
+const initialState = {
+  resources: [],
+  loading: false,
+  error: null,
+  topicId: null,
+  feedback: null,
+  newURI: null
+};
 
 describe('<TopicDashboard />', () => {
+  const mockFn = jest.fn();
   it('renders without crashing', () => {
-    shallow(<TopicDashboard />);
+    shallow(<TopicDashboard dispatch={mockFn} {...initialState} />);
```

Comment:
Why not add `dispatch: jest.fn(),` to the initial state? Other tests would also need dispatch.

Reply:
Changed so mock function is added to state